browser/ui/tabs/brave_tab_color_mixer.cc

```diff
@@ -70,21 +70,19 @@ void AddBraveTabLightThemeColorMixer(ui::ColorProvider* provider,
                                      const ui::ColorProviderKey& key) {
   ui::ColorMixer& mixer = provider->AddMixer();
 
-  static const auto kDefaultColorMap =
-      base::MakeFixedFlatMapNonConsteval<BraveColorIds, SkColor>({
-          {kColorBraveVerticalTabActiveBackground,
-           mixer.GetResultColor(kColorTabBackgroundActiveFrameActive)},
-          {kColorBraveVerticalTabInactiveBackground,
-           mixer.GetResultColor(kColorToolbar)},
-          {kColorBraveVerticalTabSeparator,
-           SkColorSetA(SK_ColorBLACK, 0.05 * 255)},
-          {kColorBraveVerticalTabNTBIconColor,
-           SkColorSetARGB(0.6 * 255, 0x1D, 0x1F, 0x25)},
-          {kColorBraveVerticalTabNTBTextColor, SkColorSetRGB(0x6B, 0x70, 0x84)},
-          {kColorBraveVerticalTabNTBShortcutTextColor,
-           SkColorSetRGB(0x85, 0x89, 0x89)},
-          {kColorBraveSplitViewTileBackground, SkColorSetRGB(0xDA, 0xDF, 0xE1)},
-      });
+  static const base::flat_map<BraveColorIds, SkColor> kDefaultColorMap = {
```

Comment:
please `base::NoDestructor`

Reply:
Updated in [3dee1cb](https://github.com/brave/brave-core/pull/22936/commits/3dee1cbed6672b5a8c42a02423a53e46bc806d7f)